packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -218,6 +218,7 @@ const NewNetwork = decorate([
                 encapsulation: encapsulation,
                 xoPifIds: pifIds,
                 encrypted,
+                mtu: +mtu,
```

Comment:
IMHO, you should explicitly handle the default value.

Reply:
in case default value `''`, `+mtu` is 0 so the XAPI puts it defaults value (1500) in the network.

Reply:
For `xo-server`'s API, the default value is not `0`, it is `undefined`.